tools/local-test/scripts/test-it.sh

```
@@ -48,6 +50,7 @@ mkdir -p $RESULT_PATH
 
 rm -rf $TMP_PATH/*
 rm -rf $RESULT_PATH/*
+rm -rf $LOGS_PATH/*
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)